schema/src/project.yaml

```diff
@@ -70,11 +139,143 @@ slots:
+    slot_uri: rfs:subWatershed
+    description: The sub-watershed associated with the project.
+  ecosystemType:
+    slot_uri: rfs:ecosystemType
+    range: EnvironmentTypeTypes
+    description: The type of ecosystem associated with the project.
+    multivalued: true
+  environmentalAuthority:
+    slot_uri: rfs:environmentalAuthority
+    range: Organization
+    description: The environmental authority associated with the project.
+  administrativeArea:
+    range: AdministrativeArea
+    description: The administrative area associated with the project.
+  offchainCreditsInfo:
+    slot_uri: rfs:offchainCreditsInfo
+    range: OffchainCreditsInfo
+    description: Information about offchain credits associated with the project.
+  projectDuration:
+    slot_uri: rfs:projectDuration
+    range: string
+    description: The duration of the project.
+  ecologicalConnectivityIndex:
+    slot_uri: rfs:ecologicalConnectivityIndex
+    range: float
+    description: The ecological connectivity index of the project.
+  socialCulturalIndex:
+    slot_uri: rfs:socialCulturalIndex
+    range: float
+    description: The social cultural index of the project.
+  managementAreas:
+    slot_uri: rfs:managementAreas
+    range: ManagementArea
+    description: The management areas associated with the project.
+    multivalued: true
+    inlined: true
+  activity:
+    # TODO: associate with regen data standards activities
+    range: ActivityType 
+    description: the activity.
+    multivalued: true
+  extent:
+    slot_uri: dcterms:extent
+    range: QuantityValue
+    description: the extent.
+  projectOperator:
+    slot_uri: rfs:projectOperator
+    range: Organization
+    description: The organization responsible for operating the project.
+  projectMonitor:
+    slot_uri: rfs:projectMonitor
+    range: Organization
+    description: The organization responsible for monitoring the project.
+  projectOwner:
+    slot_uri: rfs:projectOwner
+    range: Organization
+    description: The organization responsible for owning the project.
+  projectDeveloper:
+    slot_uri: rfs:projectDeveloper
+    range: Organization
+    description: The organization responsible for owning the project.
+  projectVerifier:
+    slot_uri: rfs:projectVerifier
+    range: Organization
+    description: The organization responsible for owning the project.
+  creditClassVersion:
+    slot_uri: rfs:creditClassVersion
+    range: string
+    description: The version of the credit class used for the project.
+  
+  
 
 enums:
   ProjectRoleTypes:
     permissible_values:
       individual:
         meaning: rfs:Individual
       organization:
-        meaning: rfs:Organization
\ No newline at end of file
+        meaning: rfs:Organization
+  MarketTypeTypes:
+    description: The type of marketplace
+    permissible_values:
+      rfs:ComplianceMarket:
+        description: a compliance market
+      rfs:VoluntaryMarket:
+        description: a voluntary market
```

Comment:
I think the `rfs:` prefix should only be included in the `meaning` property of the enum's permissible value. This way `rfs` will be expanded to the full URI from the context.  The key for each value should just be the text identifier.  See how `ProjectRoleTypes` is configured.

Reply:
done